pkg/api/handlers/types.go

```diff
@@ -215,7 +215,7 @@ func ImageToImageSummary(l *libpodImage.Image) (*entities.ImageSummary, error) {
 		Created:      l.Created().Unix(),
 		Size:         int64(*size),
 		SharedSize:   0,
-		VirtualSize:  l.VirtualSize,
```

Comment:
I think this is correct as-is, though the VirtualSize output itself may be bugged right now.

Reply:
I absolutely agree. The only reason why I did it that way is because my Go knowledge is minimal, so I couldn't figure out why it's wrong now. So I looked at what inspect command does and took it from there.